### DISCUSSION TITLE
Nouveau nom

### DISCUSSION THREAD

**Discussion by Matyu9** [Cantina-Org/cantina#79] [General]
Je voudrais changer le nom de tout les sous-projets (de chaque service).
Je voudrais que ça rentre plus en accord avec le nom du projet (Cantina) donc plus dans l'univers de Star Wars.
Voici ce que j'ai déjà pensé:
- Ouranos => Lucas (en hommage au créateur, vu que c'est l'installateur des projets)
- Néphélées => Bespin ([Planete sur la quelle la cité des nuages se situe](https://starwars.fandom.com/fr/wiki/Bespin))
- Hermes => HoloNet ([L'Holonet était un réseau de communication galactique utilisé par les citoyens de la](https://starwars.fandom.com/fr/wiki/HoloNet))
- Cerbere => Je sais pas encore
- Olympe => Coruscant ([Centre de commandement de l'ancienne république et  de l'empire impérial](https://starwars.fandom.com/fr/wiki/Coruscant#%C3%88re_de_l'Ancienne_R%C3%A9publique))
- Server Manager => Je sais pas encore

**Comment by Timtonix** [comment]
Et pourquoi pas quelque chose de plus explicite ?

**Reply by Matyu9** [reply]
Propose et on pourra voir
